src/main/scala/com/gu/subscriptions/cas/directives/ProxyDirective.scala

```diff
@@ -89,7 +89,7 @@ trait ProxyDirective extends Directives with ErrorRoute with LazyLogging {
     onSuccess(validSubscription) {
       case Some(subscription) =>
         if (activation) { subscriptionService.updateActivationDate(subscription) }
-        complete(SubscriptionExpiration(subscription.termEndDate))
+        complete(SubscriptionExpiration(subscription.termEndDate.toDateTimeAtStartOfDay()))
```

Comment:
Was this a particular bug? If the SubscriptionExpiration reads only the date (and not the time) then when the UK is in BST (and the server remains in UTC), then the subscription may terminate the day before...?